### PR TITLE
Fix else-if widget not rendering issue

### PR DESCRIPTION
**Pull request by vinothvino42**
Ticket: [#755](https://github.com/EnsembleUI/ensemble/issues/755)

### CODE REVIEW

**Review by vusters — COMMENTED** [review]
@snehmehta please review this.

File: lib/widget/conditional.dart [hunk 1]
```diff
@@ -112,7 +112,10 @@ class ConditionalState extends WidgetState<Conditional> {
       return _buildWidget(scopeManager, conditions.first);
     }
 
-    for (var i = 1; i < conditions.length - 1; i++) {
```
[the remark this answers is not in the document]
Comment (vusters) [reply lib/widget/conditional.dart:115]: @vinothvino42 as discussed please also add an integration test for this. This is very important widget.

[the remark this answers is not in the document]
Comment (vinothvino42) [reply lib/widget/conditional.dart:115]: @snehmehta Looks like your comment is not pointed to the updated code I guess. See here - https://github.com/EnsembleUI/ensemble/blob/5526c0ec883a656bd97f6e6e2007521fcc401262/lib/widget/conditional.dart#L118

Screenshot 2023-10-11 at 10 16 50 AM (2)

[the remark this answers is not in the document]
Comment (vinothvino42) [reply lib/widget/conditional.dart:115]: > @vinothvino42 as discussed please also add an integration test for this. This is very important widget.

Yeah, I'm working on that vu

**Review by snehmehta — APPROVED** [review]

File: lib/widget/conditional.dart (hunk 1 above)
[the remark this answers is not in the document]
Comment (vinothvino42) [reply lib/widget/conditional.dart:115]: > @vinothvino42 as discussed please also add an integration test for this. This is very important widget.

@vusters Can you check my PR? I added integration tests for the Conditional Widget.

**Pull request merged**